UndertaleModLib/UndertaleData.cs

```diff
@@ -289,26 +289,6 @@ public class UndertaleData : IDisposable
         /// </summary>
         public bool ShortCircuit = true;
 
-        /// <summary>
-        /// Whether the data file is from version GMS2.2.2.302
-        /// </summary>
-        public bool GMS2_2_2_302 = false;
-
-        /// <summary>
-        /// Whether the data file is from version GMS2.3
-        /// </summary>
-        public bool GMS2_3 = false;
```

Comment:
Revert this field removal, and update the description so it's clear that only this field is kept because of performance.
(see my commentary at `IsVersionAtLeast()`)